app/src/main/java/org/gdg/frisbee/android/eventseries/GdgEventListFragment.java

```diff
@@ -27,7 +28,15 @@ public static EventListFragment newInstance(String plusId) {
         fragment.setArguments(arguments);
         return fragment;
     }
-    
+
+    @Override
+    public void onActivityCreated(Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+
+        final ViewGroup.LayoutParams layoutParams = getListView().getLayoutParams();
+        layoutParams.width = (int) getResources().getDimension(R.dimen.responsive_width);
```

Comment:
It actually works :)
I was having difficulties and solving this problem with different Styles for different orientations but then I found this:
http://blog.danlew.net/2015/01/06/handling-android-resources-with-non-standard-formats/